### PR TITLE
This commit moves the development instructions to a separate file.

### PR DESCRIPTION
**Pull request by dlorenc**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,6 +59,9 @@ The error is normal because the default command is designed for child containers
 FROM openjdk
 ADD my_app_0.0.1.jar app.jar
 ```
+# Devlopment Guide
```
Comment (meltsufin) [inline README.md]: Typo: Development

File: README.md [hunk 2]
```diff
@@ -75,6 +59,9 @@ The error is normal because the default command is designed for child containers
...
+
+* See [DEVELOPING.md](DEVELOPING.md)
```
Comment (meltsufin) [inline README.md]: Maybe say, "See [instructions](DEVELOPING.md) on how to build the image from source."?

File: README.md (hunk 1 above)
Comment (dlorenc) [inline README.md]: Fixed.

File: README.md (hunk 2 above)
Comment (dlorenc) [inline README.md]: Done.

**Comment by dlorenc** [comment]
I just poked Travis and everything is passing now.

**Pull request merged**